content/blog/nodepool-upgrades/index.md

```diff
@@ -0,0 +1,161 @@
+---
+path: '/nodepool-upgrades'
+title: 'Safer Node Pool Upgrades'
```

Comment:
I would probably call this Safer Kubernetes Node Pool Upgrades for SEO. Also, I would go through this and bold some occasional sentences/fragments to help the skimmers of the world. Otherwise, looks good